liquidprompt

```diff
@@ -826,12 +828,44 @@ _lp_sb() {
     [[ -n "$1" ]] && echo -nE " $1 "
 }
 
+
+# Adds as much $3 character (e.g. spaces) between $1 and $2,
+# so as to make the resulting string the same width than the current terminal.
+_lp_fill() {
+
+    local left="$1" right="$2" space="$3"
+    [[ -z $3 ]] && space=" "
```

Comment:
This is simpler, right?
```suggestion
    local left="$1" right="$2" space="${3:-" "}"
```

Reply:
If not more readable, at least probably a little bit faster.